PowerEditor/installer/nativeLang/english.xml

```diff
@@ -1555,6 +1555,7 @@ Press the OK button to open the Find dialog or set focus on it.
 			
 If you require the backward regex searching feature, consult the user manual for instructions on enabling it."/>
 			<PrintError title="0" message="Cannot start printer document."/><!-- Use title="0" to use Windows OS default translated "Error" title. -->
+			<FindAutoChangeOfInSelectionWarning title="Search warning" message="The &quot;In selection&quot; checkbox state has been automatically modified. Please verify the search condition before performing the action."/> <!-- HowToReproduce: https://github.com/notepad-plus-plus/notepad-plus-plus/issues/14897#issuecomment-2564316224 -->
```

Comment:
Please break down the one line sentence `The "In selection" checkbox state has been automatically modified. Please verify the search condition before performing the action.` into 2 lines.